components/pkg-export-tar/src/cli.rs

```diff
@@ -0,0 +1,153 @@
+                Arg::with_name("HAB_SUP_PKG")
+                    .long("sup-pkg")
+                    .value_name("HAB_SUP_PKG")
+                    .validator(valid_ident_or_hart)
+                    .help(
+                        "Supervisor package identifier (ex: core/hab-sup) or filepath to a Habitat \
+                        artifact (ex: \
+                        /home/ore-hab-sup-0.39.1-20171118011657-x86_64-linux.hart) to install \
+                        (default: core/hab-sup)",
+                    ),
+            );
+
+        Cli { app: app }
+    }
+
+    pub fn add_builder_args(self) -> Self {
+        let app = self.app
+            .arg(
+                Arg::with_name("BLDR_URL")
+                    .long("url")
+                    .short("u")
+                    .value_name("BLDR_URL")
+                    .validator(valid_url)
+                    .help(
+                        "Install packages from Builder at the specified URL \
+                        (default: https://bldr.habitat.sh)",
+                    ),
+            )
+            .arg(
+                Arg::with_name("CHANNEL")
+                    .long("channel")
+                    .short("c")
+                    .value_name("CHANNEL")
+                    .help(
+                        "Install packages from the specified release channel (default: stable)",
+                    ),
+            )
+            .arg(
+                Arg::with_name("BASE_PKGS_BLDR_URL")
+                    .long("base-pkgs-url")
+                    .value_name("BASE_PKGS_BLDR_URL")
+                    .validator(valid_url)
+                    .help(
+                        "Install base packages from Builder at the specified URL \
+                        (default: https://bldr.habitat.sh)",
+                    ),
+            )
+            .arg(
+                Arg::with_name("BASE_PKGS_CHANNEL")
+                    .long("base-pkgs-channel")
+                    .value_name("BASE_PKGS_CHANNEL")
+                    .help(
+                        "Install base packages from the specified release channel \
+                        (default: stable)",
+                    ),
+            );
+
+        Cli { app: app }
+    }
+    pub fn add_pkg_ident_arg(self, options: PkgIdentArgOptions) -> Self {
+        let help = if options.multiple {
+            "One or more Habitat package identifiers (ex: acme/redis) and/or filepaths to a \
+            Habitat Artifact (ex: /home/acme-redis-3.0.7-21120102031201-x86_64-linux.hart)"
+        } else {
+            "A Habitat package identifier (ex: acme/redis) and/or filepath to a Habitat Artifact \
+            (ex: /home/acme-redis-3.0.7-21120102031201-x86_64-linux.hart)"
+        };
+
+        let app = self.app.arg(
+            Arg::with_name("PKG_IDENT_OR_ARTIFACT")
+                .value_name("PKG_IDENT_OR_ARTIFACT")
+                .required(true)
+                .multiple(options.multiple)
```

Comment:
I think we could probably just remove this `.multiple` call, as well as the copy of `PkgIdentArgOptions` in this crate, in the interest of keeping everything here as simple and focused as possible.

Reply:
Fixed!